### PR TITLE
Release Notes v1.73.0

### PR DESCRIPTION
**Pull request by josunect**
Release notest for 1.73

### CODE REVIEW

File: content/en/news/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,33 @@ weight: 1
 
 For additional information check our [sprint demo videos](https://www.youtube.com/channel/UCcm2NzDN_UCZKk2yYmOpc5w) and [blogs](https://medium.com/kialiproject).
 
+## v1.73.0
+Sprint Release: Sep 15, 2023
```
Comment (jmazzitelli) [inline content/en/news/release-notes.md]: We released 1.73 on August 28.

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
I'm confused about this PR - it says "1.73" with a date in the future of Sept 15, and it has issues that were just merged. Is this for 1.74? If so, we are missing 1.73 release notes. If this is for 1.73 then the notes have been generated with the wrong issues and the date should change from Sept 15 to Aug 28 in the release-notes.md

File: content/en/news/release-notes.md (hunk 1 above)
Comment (josunect) [reply content/en/news/release-notes.md]: oh! That's a wrong date - It is supposed to be for 1.73.

The release notes from 1.73 are missing, right? (That's why I created this PR)

**Comment by josunect** [comment]
> I'm confused about this PR - it says "1.73" with a date in the future of Sept 15, and it has issues that were just merged. Is this for 1.74? If so, we are missing 1.73 release notes. If this is for 1.73 then the notes have been generated with the wrong issues and the date should change from Sept 15 to Aug 28 in the release-notes.md

I used the relnotes.py script, not sure if it could be used for 1.73 (As it uses the project number but we are reusing it now)

**Comment by jmazzitelli** [comment]
@jshaughn knows the magic in that relnotes script - he can tell you how to generate for an already-released kiali

**Comment by josunect** [comment]
> @jshaughn knows the magic in that relnotes script - he can tell you how to generate for an already-released kiali

Great! Let's wait for @jshaughn then. Thank you!

**Comment by jshaughn** [comment]
Hmmm, I didn't notice that we didn't gen release notes for 1.73.  The issue here, well, at least one issue, is that we archived the "Done" column in the project board.  The release notes really need to be generated after everything is moved to Done and before archiving and then repopulating Done with stuff from the next sprint.  Let me see what I can do...

**Comment by josunect** [comment]
Maybe we could use a filter by dates in such case? (In the UI I used `is:issue closed:2023-08-04..2023-08-28`  )

**Comment by jshaughn** [comment]
Closing in favor of https://github.com/kiali/kiali.io/pull/695 and https://github.com/kiali/kiali.io/pull/696.

**Comment by jmazzitelli** [comment]
this PR can be closed - the release notes were done in these PRs:
* https://github.com/kiali/kiali.io/pull/695
* https://github.com/kiali/kiali.io/pull/696

**Pull request closed without merging**